src/csync/csync_util.cpp

```diff
@@ -198,3 +198,5 @@ bool csync_is_collision_safe_hash(const QByteArray &checksum_header)
     return checksum_header.startsWith("SHA")
         || checksum_header.startsWith("MD5:");
 }
+
+#include "moc_csync.cpp"
```

Comment:
Why do we need this?  And why here?

I guess we need this because the file does not have Q_OBJECT which is recognized by cmake.
maybe something like
```
#if 0
Q_OBJECT
#endif 
```

would work...


But anyway, if including this file work as well, that's ok too.

Reply:
Have to fix windows builds :)

Reply:
Sigh...
To use that on windows we need qt 5.14...
https://doc.qt.io/qt-5/qobject.html#Q_NAMESPACE_EXPORT

Reply:
before, you can still do 
`CSYNC_EXPORT Q_NAMESPACE`
with the same effect

Reply:
Your right so there was no reason at all to add jet another macro...